app/models/html_publication.rb

```diff
@@ -1,5 +1,6 @@
 class HtmlPublication < ContentItem
   include Political
+  include NationalApplicability
```

Comment:
Shared example for this could be added at the same time:
https://github.com/alphagov/frontend/blob/main/spec/models/call_for_evidence_spec.rb#L10